### PR TITLE
Desktop: Change template prompt to sans and sort templates

### PR DESCRIPTION
**Pull request by CalebJohn**
Addresses [this topic](https://discourse.joplinapp.org/t/minor-issues-with-templates/3194) in the forum.

Note: I wasn't able to reproduce his second issue (sort order). But I put some sorting code in, hopefully that will fix it.

### CODE REVIEW

File: ReactNativeClient/lib/TemplateUtils.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ TemplateUtils.loadTemplates = async function(filePath) {
 		});
 	}
 
-	return templates;
+	return templates.sort((a, b) => { return a.label.localeCompare(b.label); });
```
Comment (laurent22) [inline ReactNativeClient/lib/TemplateUtils.js]: Shouldn't we sort in a case insensitive way? (not sure if localeCompare does this)

Comment (CalebJohn) [reply ReactNativeClient/lib/TemplateUtils.js]: You're probably right, I just copied this function from the tag sorting. I'll look into a better method for sorting.

Comment (CalebJohn) [reply ReactNativeClient/lib/TemplateUtils.js]: I changed this by adding the sensitivity option.

**Comment by CalebJohn** [comment]
Okay should be good to ship now.

**Comment by laurent22** [comment]
Looks good now, thanks @CalebJohn!

**Pull request merged**